service.subsmangler/resources/lib/pysubs2/microdvd.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import unicode_literals, print_function
```

Comment:
At the moment, the included version of pysubs2 is a slightly modified (added .tmp conversion routine) fork of the original pysubs2 library written by T. Karabela. 
If it is OK, As long as my PR is not pulled in to the pysubs2 upstream branch and released as a new version, I would prefer to keep a static copy of it to avoid differences between original pysubs2 and the script.module.

Reply:
Its fine as is, it was just a suggestion. Main issue is the entrypoint complexity which could be improved with minimal effort

Reply:
I have updated the commit. Please see if it is ok.